src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -87,6 +88,234 @@ protected override Conversion GetInterpolatedStringConversion(BoundUnconvertedIn
                 ? Conversion.InterpolatedString : Conversion.NoConversion;
         }
 
+#nullable  enable
+        public override bool IsApplicableInterpolatedStringBuilderType(BoundUnconvertedInterpolatedString source, TypeSymbol builderType, BindingDiagnosticBag diagnostics, out ImmutableArray<MethodArgumentInfo> builderArguments)
```

Comment:
>builderArguments [](start = 211, length = 16)

Are these really arguments? I think we need to come up with a name that more accurately reflects the nature of data.